container/core.py

```diff
@@ -32,6 +32,7 @@
 
 REMOVE_HTTP = re.compile('^https?://')
 DEFAULT_CONDUCTOR_BASE = 'centos:7'
+DEFAULT_DEPLOYMENT_PATH = '~/ansible-deployment/'
```

Comment:
I'm thinking this would be better to default to a relative path, basically `os.path.join(config.base_path, 'ansible-deployment/')` how does that sound?